src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -434,6 +434,7 @@ + (void)initialize {
   // Guarantees the code in {} block is invoked only once. See ref at:
   // https://developer.apple.com/documentation/objectivec/nsobject/1418639-initialize?language=objc
   if (self == [GRPCCall self]) {
+    setenv(kCFStreamVarName, "1", 1);
```

Comment:
is `kCFStreamVarName` used/specified anywhere?

Reply:
Yes; it was already in the beginning of this file.